.github/workflows/build.yml

```diff
@@ -37,11 +37,20 @@ jobs:
         with:
           command: clippy
           args: -- -D warnings
+      - uses: actions-rs/cargo@v1
```

Comment:
replaced 'check' with 'build' and moved it to the end, as it's the most time-consuming